### PR TITLE
Generic CMK check

### PR DESCRIPTION
**Pull request by JamesWoolfenden**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
Looks for the same key attribute in a large number of resources, rather than write the same check again,

### CODE REVIEW

**Comment by JamesWoolfenden** [comment]
Not sure what you want me to do here?

File: checkov/terraform/checks/resource/aws/ResourceEncyyptedWithCMK.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from checkov.common.models.consts import ANY_VALUE
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.enums import CheckCategories
+
+
+class ResourceEncryptedWithCMK(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure resource is encrypted by KMS"
```
Comment (mikeurbanski1) [inline checkov/terraform/checks/resource/aws/ResourceEncyyptedWithCMK.py:8]: Should specifically mention CMK

Comment (JamesWoolfenden) [reply checkov/terraform/checks/resource/aws/ResourceEncyyptedWithCMK.py]: Updated

**Review by nimrodkor — APPROVED** [review]
Ta! @JamesWoolfenden

**Pull request merged**